CircleMsg[] circle_vec

int32 index

# including the position from where it is broadcasting the message
float32 robot_x
float32 robot_y
float32 robot_z